packages/synapse-react-client/src/components/ProjectStorage/ProjectStorage.stories.ts

```diff
@@ -0,0 +1,77 @@
+import { Meta, StoryObj } from '@storybook/react'
+import ProjectDataAvailability from './ProjectDataAvailability'
+import { MOCK_REPO_ORIGIN } from 'src/utils/functions/getEndpoint'
+import { getUserProfileHandlers } from 'src/mocks/msw/handlers/userProfileHandlers'
+import { getEntityHandlers } from 'src/mocks/msw/handlers/entityHandlers'
+import { getProjectStorageHandlers } from 'src/mocks/msw/handlers/projectStorageHandlers'
+import {
+  OVER_LIMIT_PROJECT_ID,
+  UNDER_LIMIT_PROJECT_ID,
+} from 'src/mocks/projectStorage/mockProjectStorageLimits'
+
+const meta = {
+  title: 'Synapse/ProjectStorage',
+  component: ProjectDataAvailability,
+  parameters: {
+    chromatic: { viewports: [600, 1200] },
+  },
+  argTypes: {
+    isAuthenticated: {
+      type: 'boolean',
+    },
+  },
+  args: {
+    isAuthenticated: true,
+  },
+} satisfies Meta
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const ProjectDataUnderLimit: Story = {
+  args: {
+    projectId: UNDER_LIMIT_PROJECT_ID,
+    sx: { backgroundColor: '#375574' },
```

Comment:
Note, I'm setting the background color in the story (not in the component)

Reply:
You could instead wrap the story in a `decorator` that applies the background (e.g. EnumFacetFilter.stories.tsx)

Then remove the sx prop if this is the only place you plan to use it.

Reply:
I initially implemented a decorator, but thought it would be more flexible if this component supported SxProps!  Happy to change to a decorator if you think it's more appropriate